.github/workflows/e2e.yml

```diff
@@ -207,11 +207,9 @@ jobs:
         check_run_annotations: all tests
         comment_mode: create new
 
-    - name: Archive logs
-      uses: papeloto/action-zip@v1
+    - uses: montudor/action-zip@v1
       with:
-        files: test/e2e/reports
-        dest: reports.zip
+        args: zip -qq -r reports.zip test/e2e/reports
 
     - name: Upload report to artifacts
```

Comment:
What happens, if we try to upload a folder here?

Reply:
https://github.com/actions/upload-artifact#zipped-artifact-downloads

Reply:
thank you